CHANGELOG.md

```diff
@@ -275,7 +281,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - Initial version.
 
-[unreleased]: https://github.com/dividab/tsconfig-paths/compare/3.8.0...master
+[unreleased]: https://github.com/dividab/tsconfig-paths/compare/v3.9.0...master
+[3.9.0]: https://github.com/dividab/tsconfig-paths/compare/v3.8.0...v3.9.0
```

Comment:
FYI: all the links below are broken

Reply:
I can fix that after merging. Also I usually put the links inline now instead of at the bottom, that way it's easier to remember to add them.